recipe/meta.yaml

```
@@ -1,25 +1,31 @@
-{% set name = "cni" %}
-{% set version = "0.6.0" %}
+{% set import_path = "github.com/containernetworking/cni" %}
+{% set version = "0.7.1" %}
+
+{% set name = import_path.split('/')[-1] %}
+{% set pkg_src = ('src/' + import_path).replace("/", "\\" if win else "/") %}
```

lets use os.sep here, it looks cleaner.
```suggestion
{% set pkg_src = ('src/' + import_path).replace("/", os.sep) %}
```